03_letters/pfannerj/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var mainout io.Writer = os.Stdout
+
+func main() {
+
+	fmt.Fprintf(mainout, strings.Join(sortLetters(letters("aba")), "\n"))
+
+}
+
+func letters(s string) map[rune]int {
+
+	mapOfRunes := map[rune]int{}
+	allRunes := []rune(s)
+	count := 0
+	for i := range allRunes {
+		count = mapOfRunes[allRunes[i]]
+		mapOfRunes[allRunes[i]] = count + 1
+	}
+	return mapOfRunes
+}
+
+func sortLetters(m map[rune]int) []string {
+
+	lettersCount := make([]string, len(m))
+	i := 0
+	for thisRune := range m {
```

Comment:
please consider sth like `for i, thisRune := range m {`

Reply:
I tried out that but it doesn't get me what I need... if I say  <for thisRune, i := range m {> then <i> will be the count and not the index of the slice.  I've defined and incremented <i> separately so that I can use it as an index to the new lettersCount slice and insert the strings before sorting it... will leave this one as is...

Reply:
awww yes, m is map. sorry for this.